packages/suite/src/views/wallet/tokens/common/TokensTable/TokensTable.tsx

```diff
@@ -0,0 +1,130 @@
+`;
+
+interface TokensTableProps {
+    account: Account;
+    tokensWithBalance: EnhancedTokenInfo[];
+    tokensWithoutBalance: EnhancedTokenInfo[];
+    network: NetworkCompatible;
+    tokenStatusType: TokenManagementAction;
+    hideRates?: boolean;
+    searchQuery?: string;
+    isUnverifiedTable?: boolean;
+}
+
+export const TokensTable = ({
+    account,
+    tokensWithBalance,
+    tokensWithoutBalance,
+    network,
+    tokenStatusType,
+    hideRates,
+    searchQuery,
+    isUnverifiedTable,
+}: TokensTableProps) => {
+    const [isZeroBalanceOpen, setIsZeroBalanceOpen] = useState(false);
+
+    return (
+        <Card paddingType="none">
+            {tokensWithBalance.length === 0 && tokensWithoutBalance.length === 0 && searchQuery ? (
+                <Paragraph
+                    typographyStyle="highlight"
+                    margin={{ top: spacings.xxl, bottom: spacings.xxl }}
+                    align="center"
+                >
+                    <Translation id="TR_NO_SEARCH_RESULTS" />
+                </Paragraph>
+            ) : (
+                <Table margin={{ top: spacings.xs, bottom: spacings.xs }} colWidths={['250px']}>
+                    <Table.Header>
+                        <Table.Row>
+                            <Table.Cell>
+                                <Translation id="TR_TOKEN" />
+                            </Table.Cell>
+                            <Table.Cell colspan={hideRates ? 2 : 1}>
+                                <Translation id="TR_VALUES" />
+                            </Table.Cell>
+                            {!hideRates && (
+                                <>
+                                    <Table.Cell align="right">
+                                        <Translation id="TR_EXCHANGE_RATE" />
+                                    </Table.Cell>
+                                    <Table.Cell colspan={2}>
+                                        <Translation id="TR_7D_CHANGE" />
+                                    </Table.Cell>
+                                </>
+                            )}
+                        </Table.Row>
+                    </Table.Header>
+                    <Table.Body>
+                        {tokensWithBalance.map(token => (
+                            <TokenRow
+                                key={token.symbol}
+                                token={token}
+                                account={account}
+                                network={network}
+                                tokenStatusType={tokenStatusType}
+                                isUnverifiedTable={isUnverifiedTable}
+                                hideRates={hideRates}
+                            />
+                        ))}
+                        {tokensWithoutBalance.length !== 0 && (
+                            <>
+                                <Table.Row>
+                                    <Table.Cell colspan={hideRates ? 3 : 5}>
```

Comment:
I think to make this sticky, it has to have colspan="1" but the text has to overflow over columns

Reply:
Yes you're right, I've experimented with this and it also works with colspan 2 without overflowing